include/libu64/relocation.h

```diff
@@ -0,0 +1,40 @@
+#ifndef RELOCATION_H
+#define RELOCATION_H
```

Comment:
This file is named loadfragment.h in MM and in my existing branch.

Reply:
I too think relocation.h is a better name

Reply:
I'd argue that relocation is too narrow a scope. `Overlay_Relocate` / `relocation.c` is the only bit that actually performs relocation, whereas `Overlay_Load` is doing more than just relocating the symbol data.

Reply:
Very fair actually. I would like to do "overlay.h" then. This is in line with the Overlay_ system name the functions already have, and is more common terminology than calling them "fragments"

Reply:
i guess.